.ruby-version

```diff
@@ -1 +1 @@
-2.1.1
+2.5.3
```

Comment:
Things break in development since the version is so old (something to do with rubygems). Using ruby `2.5.3` which is what Bladerunner uses seems to not cause any issues 🤷‍♂